next-sitemap.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
```

Comment:
this looks cool, very convenient. just wonder, if there's no mistake in autogenerated files, I see 
https://olas-website-git-precious-seo-issues-autonolas.vercel.app/robots.txt shows 
```
# Sitemaps
Sitemap: https://olas.network/sitemap.xml
Sitemap: https://olas.network/server-sitemap.xml
```

however https://olas-website-git-precious-seo-issues-autonolas.vercel.app/sitemap.xml shows the list of sitemaps 
```
<sitemap>
  <loc>https://olas.network/sitemap-0.xml</loc>
</sitemap>
<sitemap>
  <loc>https://olas.network/server-sitemap.xml</loc>
</sitemap>
```
so the `server-sitemap.xml` is mentioned twice, and it returns 500 error.
Can we just simply have `sitemap-0.xml`'s content in `sitemap.xml` and remove `server-sitemap` as I don't think we need it

Reply:
That's what we've been trying to figure out how to do :/ It generates the static pages but it's all inside sitemap-0.xml for some reason